example/50-seed.yaml

```diff
@@ -26,6 +26,12 @@ spec:
       namespace: garden
   dns:
     ingressDomain: dev.my-seed.example.com
+#   nginxIngress:
```

Comment:
Let's rename this to `ingress` or `ingressController` so that we can switch internally if we want and do not hard-code the nginx-ingress.